libtcmu_log.c

```diff
@@ -548,6 +582,14 @@ static void *log_thread_start(void *arg)
 	}
 	pthread_mutex_unlock(&tcmu_logbuf->lock);
 
+	pthread_mutex_lock(&log_tmpbuf_lock);
+	darray_foreach(lb, log_tmpbuf)
+		log_output(lb->pri, lb->msg, true);
+	darray_free(log_tmpbuf);
+	pthread_mutex_unlock(&log_tmpbuf_lock);
+
+	tcmu_info("Log thread finishes initializing!\n");
```

Comment:
This should be a debug message. Also fix up the wording:

tcmu_dbg("Log thread finished initializing!\n");

Reply:
Yeah, Will update this.